portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/upgrades/UpgradeReport.testcase

```diff
@@ -631,6 +647,33 @@ definition {
 		}
 	}
 
+	@description = "LPS-181594: The message with the  upgrade report file path is printed in the logs"
+	@priority = 4
+	test UpgradeReportFilePathMessageLogged {
+		task ("Look for Liferay Home") {
+			var liferayHome = PropsUtil.get("liferay.home.dir.name");
+		}
+
+		task ("Format local liferay home path to match the one printed in the logs") {
+			var localLiferayHomeInitialFilePath = StringUtil.extractFirst(${liferayHome}, "/liferay-portal");
+			var localLiferayHome = StringUtil.extractLast(${liferayHome}, "../");
+
+			var localLiferayHome = "${localLiferayHomeInitialFilePath}/${localLiferayHome}";
+		}
+
+		task ("Look for the upgrade logs") {
+			var fileContentLog = FileUtil.read("${liferayHome}/tools/portal-tools-db-upgrade-client/logs/upgrade.log");
+		}
```

Comment:
Thats cause the format that the local is returning Liferay Home in the Log is different from the format we got in the test (locally only).
Example - 
**Caught testing :** /home/me/dev/`projects/liferay-portal/../bundles`
**Printed in the log:** [java] 2023-04-18 20:03:17.788 INFO [main][UpgradeReport:730] Upgrade report generated in /home/me/dev/`projects/bundles`/tools/portal-tools-db-upgrade -client/./reports/upgrade_report.info

And this ends up disturbing the assertion of the message locally, in the end the difference is only due to the assertion, but the path is the same

Reply:
@lucasperj I suspect this happens locally since the default app.server.parent.dir has the relative path.

I think it is better we fix at a deeper level.

It gets set for the tests here: https://github.com/liferay/liferay-portal/blob/b7d0025e53d2a904baa35a4407c765129a19720e/build-test.xml#L11988

Instead of passing in the original ${liferay.home}, convert it to an absoluate path first
```
<property location="${liferay.home}" name="liferay.home.absolute" />
```